docs/sources/mimir/operators-guide/mimir-runbooks/_index.md

```diff
@@ -118,6 +118,7 @@ How to **fix** it:
 
 This alert fires when the `max_tenants` per ingester instance limit is enabled and the actual number of tenants in an ingester is reaching the limit. Once the limit is reached, writes to the ingester will fail (5xx) for new tenants, while they will continue to succeed for previously existing ones.
 
+The per-tenant memory utilisation in ingesters is made up mostly by allocations for TSDB stripes and chunk writer buffers. The size of these allocations is controlled by `-blocks-storage.tsdb.stripe-size` (default 16KiB) and `-blocks-storage.tsdb.head-chunks-write-buffer-size-bytes` (default 4MiB), respectively.
```

Comment:
newline before "in case of emergency" section:

```suggestion
The per-tenant memory utilisation in ingesters is made up mostly by allocations for TSDB stripes and chunk writer buffers. The size of these allocations is controlled by `-blocks-storage.tsdb.stripe-size` (default 16KiB) and `-blocks-storage.tsdb.head-chunks-write-buffer-size-bytes` (default 4MiB), respectively.

```